src/components/policy/policy_regular/test/vehicle_data_item_type_test.cc

```diff
@@ -156,7 +157,8 @@ TEST_F(VehicleDataItemTypeTest, Initialize_Failed) {
 }
 
 TEST_F(VehicleDataItemTypeTest, CheckConvertFromJsonToVehicleDataItem_Success) {
-  reader_.parse(GetFullJsonString(), json_);
+  std::string str = GetFullJsonString();
```

Comment:
@ydementieiev const?

Reply:
@AByzhynar done in [1345b2e](https://github.com/smartdevicelink/sdl_core/pull/3155/commits/1345b2e4f52415e896a5fa3d232aa391c69751bb)